domain/schema/model/sql/0022-resource.sql

```diff
@@ -0,0 +1,108 @@
+CREATE TABLE resource_origin_kind (
+    id INT PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE UNIQUE INDEX idx_resource_origin_name
+ON resource_origin_kind (name);
+
+INSERT INTO resource_origin_kind VALUES
+(0, 'uploaded'),
+(1, 'store');
+
+CREATE TABLE resource_state (
+    id INT PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE UNIQUE INDEX idx_resource_state
+ON resource_state (name);
+
+INSERT INTO resource_state VALUES
+(0, 'available'),
+(1, 'potential');
```

Comment:
Not sure about the word potential here, any context?

Reply:
current/optional?

Reply:
Ah, right that makes more sense now it's explained. Can we add a comment explaining where it comes from? Fine with the name now.

Reply:
Commented the table with the above description.